dashboard/client/src/service/log.ts

```diff
@@ -49,3 +49,15 @@ export const getLogDetail = async (url: string) => {
 
   return rsp.data as string;
 };
+
+export const getStateApiDownloadLogUrl = (nodeId: string, fileName: string) =>
+  `api/v0/logs/file?node_id=${nodeId}&filename=${fileName}&lines=-1`;
```

Comment:
I am adding support for 
`api/v0/logs/file?job_id=<submit_job_id>&lines=-1` as part of effort to support job log as well. But this also looks fine.